lib/lbt/bundle/AutoSplitter.js

```diff
@@ -74,6 +74,25 @@ class AutoSplitter {
 					totalSize += "sap.ui.requireSync('');".length + toRequireJSName(module).length;
 				});
 				break;
+			case SectionType.DepCache:
+				moduleSizes["__depCacheSize"] = "sap.ui.loader.config({depCacheUI5:{}});".length;
```

Comment:
Also, don't we need to add the length to `totalSize` here as well (like in line 91)

Reply:
Ah, yes! Thanks a lot!